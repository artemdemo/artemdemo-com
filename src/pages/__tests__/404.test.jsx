import React from 'react';
import renderer from 'react-test-renderer';
import Page404 from '../404';

jest.mock('../../components/Layout/Layout');
jest.mock('../../components/Container/Container');

describe('404', () => {
    it('should render', () => {
        const tree = renderer.create(
            <Page404 />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
