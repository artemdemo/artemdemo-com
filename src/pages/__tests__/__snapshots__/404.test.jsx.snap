// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 should render 1`] = `
<div
  data-mock="Container"
>
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</div>
`;
